SharedPackages/VPN/Sources/VPN/Subscription/VPNSubscriptionStatusPixel.swift

```diff
@@ -22,44 +22,53 @@ import Subscription
 public enum VPNSubscriptionStatusPixel: PixelKitEventV2, PixelKitEventWithCustomPrefix {
     case vpnFeatureEnabled(isSubscriptionActive: Bool?,
                     isAuthV2Enabled: Bool,
-                    source: Source)
+                    trigger: Trigger)
     case vpnFeatureDisabled(isSubscriptionActive: Bool?,
                      isAuthV2Enabled: Bool,
-                     source: Source)
+                     trigger: Trigger)
     case signedIn(isSubscriptionActive: Bool?,
                   isAuthV2Enabled: Bool,
-                  source: Source)
+                  trigger: Trigger)
     case signedOut(isSubscriptionActive: Bool?,
                    isAuthV2Enabled: Bool,
-                   source: Source)
+                   trigger: Trigger)
 
-    public enum Source {
-        case clientCheck(sourceObject: Any?)
+    public enum Trigger {
+        case clientCheck
+#if os(macOS)
+        case clientCheckOnWake
+#elseif os(iOS)
+        case clientForegrounded
```

Comment:
On macOS we run this check on wake, but iOS doesn't handle wake events.  The closest thing is checking on foreground.